SECURITY.md

```diff
@@ -261,11 +261,31 @@ distribution should:
       marketing copy, etc.) that it is built on top of Envoy. E.g.,
       "SuperAwesomeCloudProvider's Envoy as a Service (EaaS)". An infrastructure
       service that uses Envoy for a product but does not publicly say they are
-      using Envoy does not qualify. This is essentially IaaS or PaaS, if you use
-      Envoy to support a SaaS, e.g. "SuperAwesomeCatVideoService", this does not
-      qualify.
-2. Have a user or customer base not limited to your own organization. We will use the size
-   of the user or customer base as part of the criteria to determine
+      using Envoy does not *generally* qualify (see option 3 that follows). This is essentially IaaS
+      or PaaS. If you use Envoy to support a SaaS, e.g. "SuperAwesomeCatVideoService", this does not
+      *generally* qualify.
+
+   OR
+
+   3. An end user of Envoy that satisfies the following requirements:
+       1. Is "well known" to the Envoy community. Being "well known" is fully subjective and
+          determined by the Envoy maintainers and security team. Becoming "well known" would
+          generally be achieved by activities such as: PR contributions, either code or
+          documentation; helping other end users on Slack, GitHub, and the mailing lists; speaking
+          about use of Envoy at conferences; writing about use of Envoy in blog posts; sponsoring
+          Envoy conferences, meetups, and other activities; etc. This is a more strict variant of
+          item 5 below.
+       2. Is of sufficient size, scale, and brand prominence to make your inclusion on the list
```

Comment:
I'm hesitant to mention brand prominence; our goal is to protect the maximum number of Envoy users (including those who depend on Envoy deployments). The marketing budget of an organization shouldn't be a signal here IMHO.

Reply:
My reasoning for including this is that we have already said that this whole admission path is completely subjective, and whether we say it or not, I find it impossible to believe that brand prominence is not going to play a role in that decision, so I decided to be honest about it. 

With that said I don't feel very strongly about whether to include it in the doc so would like to hear more opinions on this.

Reply:
What about couching this in terms of our end goals, so instead of brand prominence, we can say "user impact", "community impact" or simply "impact"?

Reply:
Sure sounds good.